source/frontend/src/features/leases/add/RenewalSubForm.tsx

```diff
@@ -0,0 +1,113 @@
+import { FieldArray } from 'formik';
+import { FormikProps } from 'formik/dist/types';
+import { Col, Row } from 'react-bootstrap';
+
+import { LinkButton, RemoveButton } from '@/components/common/buttons';
+import { FastDatePicker, TextArea } from '@/components/common/form';
+import { YesNoSelect } from '@/components/common/form/YesNoSelect';
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { getDeleteModalProps, useModalContext } from '@/hooks/useModalContext';
+
+import { FormLeaseRenewal, LeaseFormModel } from '../models';
+
+export interface IRenewalSubFormProps {
+  formikProps: FormikProps<LeaseFormModel>;
+}
+
+export const RenewalSubForm: React.FunctionComponent<IRenewalSubFormProps> = ({ formikProps }) => {
+  const { values } = formikProps;
+  const { setModalContent, setDisplayModal } = useModalContext();
+
+  const fieldName = 'renewals';
+
+  return (
+    <Section header="Renewal Option">
+      <FieldArray
+        name={fieldName}
+        render={arrayHelpers => (
+          <>
+            {values.renewals.map((renewal, index) => (
+              <Row key={index}>
+                <Col>
+                  <Section header={`Renewal ${index + 1}`} noPadding>
+                    <Row>
+                      <Col xs="7">
+                        <SectionField label="Excercised?" labelWidth="6" required>
+                          <YesNoSelect field={`${fieldName}.${index}.isExercised`} notNullable />
+                        </SectionField>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <Col xs="7">
+                        <SectionField
+                          label="Commencement"
```

Comment:
tooltips missing for commencement, expiry on read/edit.

Reply:
Pretty odd. I remember adding them. Anyhow, they are there now.